stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/viewholder/internal/CustomAttachmentsViewHolder.kt

```diff
@@ -122,7 +123,11 @@ internal class CustomAttachmentsViewHolder(
                     container.threadClickListener.onThreadClick(data.message)
                 }
                 root.setOnLongClickListener {
-                    container.messageLongClickListener.onMessageLongClick(data.message)
+                    if (data.message.isModerationFailed()) {
```

Comment:
Instead of doing this here and repeating it for every single attachment and holder type - what about just hooking this into the default longClick handler and checking there? That way we can just update 1 place and make sure all of our holders handle the bounce properly.

Reply:
@TGazica